packages/aws-cdk-lib/aws-stepfunctions-tasks/README.md

```diff
@@ -386,6 +386,46 @@ const task = new tasks.CodeBuildStartBuild(this, 'Task', {
 });
 ```
 
+### StartBuildBatch
+
+[StartBuildBatch](https://docs.aws.amazon.com/codebuild/latest/APIReference/API_StartBuildBatch.html) starts a CodeBuild Project by Project Name.
+It is necessary to enable the batch build feature in the CodeBuild project.
+
+```ts
+import * as codebuild from 'aws-cdk-lib/aws-codebuild';
+
+const project = new codebuild.Project(this, 'Project', {
+  projectName: 'MyTestProject',
+  buildSpec: codebuild.BuildSpec.fromObjectToYaml({
+    version: 0.2,
+    batch: {
+      'build-list': [
+        {
+          identifier: 'id',
+          buildspec: 'version: 0.2\nphases:\n  build:\n    commands:\n      - echo "Hello, from small!"',
+        },
+      ],
+    },
+  }),
+});
+const buildconfig = project.enableBatchBuilds();
+
+if (buildconfig == null) {
+  throw new Error('Batch builds not enabled');
+}
```

Comment:
```suggestion
project.enableBatchBuilds();
```
We should provide a fully working example and explain those scenarios where `enableBatchBuilds()` returns `undefined` via a comment or note below.

Reply:
I realized that `Project.enableBatchBuild()` never return `undefined`.